.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
     rev: 5.10.1
     hooks:
     -   id: isort
-# -   repo: https://github.com/PyCQA/flake8
-#     rev: 4.0.1
-#     hooks:
-#     -   id: flake8
+-   repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+    -   id: flake8
```

Comment:
By the way, adding new hooks for `pre-commit` is as easy as updating this file with a new repo and hook id. Next time the pre-commit tool is triggered, it'll fetch any missing packages and run them.